types/list-stream/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for list-stream 1.0
+// Project: https://github.com/rvagg/list-stream
+// Definitions by: IanStorm <https://github.com/IanStorm>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Duplex, DuplexOptions } from "stream";
+
+declare namespace ListStream {
```

Comment:
I don't think you need this `namespace`.

Reply:
@uniqueiniquity Thanks for checking my Pull Request, and thanks for the feedback.

While trying to fix that issue, I actually came a cross some bigger mistake that I have done while creating the definition. However, with the latest commit (f6a44f54c73a0b255e7b36a1953824e518165996) the definition should be proper. Sorry that it is a major change.